src/util/GridFilterTemplateUtil.js

```diff
@@ -0,0 +1,21 @@
+import { Dropdown } from "primereact/dropdown";
+import { ENVIROMENT_LIST, INSCRIPTIONS_STATUS, PRODUCT_LIST, SERVICE_LIST, 
+    TRANSACTION_STATUS_LIST, TRANSACTION_TBK_REFUND_LIST, TRANSACTION_TBK_STATUS_LIST } from "./TbkConstans";
+
+export const createFilerDropdown = (list) => {
```

Comment:
this should be createFilterDropdown?

Reply:
It is only for grid columns with filters